File: Cosmos-Android/app/src/main/java/wannabit/io/cosmostaion/model/kava/HardInterestRate.java
package wannabit.io.cosmostaion.model.kava;

import com.google.gson.annotations.SerializedName;

public class HardInterestRate {
    @SerializedName("denom")
    public String denom;

    @SerializedName("supply_interest_rate")
    public String supply_interest_rate;

    @SerializedName("borrow_interest_rate")
    public String borrow_interest_rate;
}

File: Cosmos-IOS/Cosmostation/Cosmostation/Model/KavaModel/HardInterestRate.swift
//
//  HardInterestRate.swift
//  Cosmostation
//
//  Created by 정용주 on 2021/03/04.
//  Copyright © 2021 wannabit. All rights reserved.
//

import Foundation

public struct HardInterestRate {
    var denom: String?
    var supply_interest_rate: String?
    var borrow_interest_rate: String?
    
    init(_ dictionary: NSDictionary) {
        self.denom = dictionary["denom"] as? String
        self.supply_interest_rate = dictionary["supply_interest_rate"] as? String
        self.borrow_interest_rate = dictionary["borrow_interest_rate"] as? String
    }
}
